src/pages/Signup/Signup.js

```diff
@@ -1,8 +1,184 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { chkEmail, chkPwd, chkName, chkPhone } from './Validation';
+import './Signup.scss';
+
+class Signup extends React.Component {
+  emailRef = React.createRef();
+  pwRef = React.createRef();
+
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    phone: '',
+  };
+
+  emailInput = email => {
+    this.setState({
+      email: email.target.value,
+    });
+  };
+
+  passwordInput = pw => {
+    this.setState({
+      pw: pw.target.value,
+    });
+  };
+
+  nameInput = name => {
+    this.setState({
+      name: name.target.value,
+    });
+  };
+
+  phoneInput = phone => {
+    this.setState({
+      phone: phone.target.value,
+    });
+  };
+
+  checkAll = e => {
+    if (chkName(this.state.name) === false) {
+      alert('이름을 입력해주세요.');
+    }
+
+    if (chkEmail(this.state.email) === false) {
+      alert('이메일 형식이 유효하지 않습니다.');
+      this.emailRef.current.value = '';
+    }
+
+    if (chkPwd(this.state.pw) === false) {
+      alert('영문,숫자를 혼합하여 6~12자 이내로 만들어주세요.');
+      this.pwRef.current.value = '';
+    }
+
+    if (chkPhone(this.state.phone) === false) {
+      alert('번호를 입력해주세요.');
+    }
+
+    if (
+      chkName(this.state.name) &&
+      chkEmail(this.state.email) &&
+      chkPwd(this.state.pw) &&
+      chkPhone(this.state.phone) === true
```

Comment:
```suggestion
      chkName(this.state.name) &&
      chkEmail(this.state.email) &&
      chkPwd(this.state.pw) &&
      chkPhone(this.state.phone)
```
chkPhone 자체가 boolean을 리턴하는 것 같은데 === true는 할 필요가 없어보입니다!

Reply:
수정 완료!!! 🙆🏻‍♀️